src/Typed/.gitignore

```diff
@@ -0,0 +1,3 @@
+vendor
+composer.lock
+.php_cs.cache
```

Comment:
Remove this: this is a file that should be ignored globally if you need to ignore it.